Bottom: ea178366eee14b5550d35c155af48676d8a64d8a
Top:    a91b1ac581031dbebf42bc43be03f6b574a8d99a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-05 15:37:32 -0500

Refresh of worker-assets-role

---
diff --git a/roles/buildbot-worker-assets/tasks/main.yml b/roles/buildbot-worker-assets/tasks/main.yml
index df9bda29..0d698a80 100644
--- a/roles/buildbot-worker-assets/tasks/main.yml
+++ b/roles/buildbot-worker-assets/tasks/main.yml
@@ -1,7 +1,7 @@
 ---
 - name: Start asset server
   docker_container:
-    name: builder-assets
+    name: buildbot-assets
     image: "httpd:alpine"
     volumes:
     - "/mnt/assets:/usr/local/apache2/htdocs/:ro"
@@ -11,6 +11,6 @@
 
 - set_fact:
     buildbot_asset_host: "http://\
-    {{asset_server.ansible_facts.docker_container.NetworkSettings.IPAddress}}\
+    {{asset_server.ansible_facts.docker_container.NetworkSettings.Gateway}}\
     :{{asset_server.ansible_facts.docker_container.NetworkSettings.Ports['80/tcp'][0].HostPort}}\
     "
